### DISCUSSION TITLE
Add feature to convert files with FFmpeg ('non-free' version)

### DISCUSSION THREAD

**Discussion by trimechee** [setvisible/DownZemAll#192] [Ideas]
Hi,  i found a "nonfree" ffmpeg that seems to overcome some of the limitations of the official classic ffmpeg.....maybe be it can help the project :

Automated nonfree builds of FFmpeg for Windows using GitHub Actions:

https://github.com/AnimMouse/ffmpeg-autobuild
https://www.animmouse.com/projects/ffmpeg-autobuild/

other builds of FFmpeg built by others :

https://www.animmouse.com/p/ffmpeg-binaries/

**Comment by setvisible** [comment]
Currently ArrowDL doesn't offer conversion of downloaded files into another format with FFmpeg. 

I'll close other tickets related to FFmpeg and will keep this one as improvement request.

Regards

**Comment by trimechee** [comment]
Great :)

**Comment by setvisible** [comment]
Hi

Due to limited time and resource, it's not planned currently. However anyone who wants to implement and propose a merge request is welcome.

Cheers
